### PR TITLE
 separate out proxy-shape translator into separate plugin

### PR DESCRIPTION
**Pull request by pmolodo**
## Description (this won't be part of the changelog)

So, we noticed that when trying to build a stock version of AL_USDMaya, we got some runtime linking errors, and the problems were with our former pr/proxyShape_translator.  This fixes those errors, and also completely separates and sandboxes the entire translator, so the main plugin does not need to link against the pixar usdMaya libraries at all.

This PR is broken into two commits - the first gets everything working for building against usd-0.8.5, and the second makes changes needed for building against usd-0.18.09

### Changed
- Made translatorProxyShape into a completely separate plugin, AL_USDMayaPxrTranslators
- This sandboxes the main AL_USDMayaPlugin from needing to link against any pixar maya libraries (ie, usdMaya)
- Also added a cmake option to disable building of the plugin entirely, BUILD_USDMAYA_PXR_TRANSLATORS
- Updated FindUSD.cmake to enable finding of usdMaya

### Fixed
- Fixed some runtime linkage problems with the main AL_USDMayaPlugin.so and usdMaya.so

## Checklist (Please do not remove this line)
- [X] Make sure the Title and Description of the PR make sense and  provide sufficient context for your work
- [X] Do any added files have the correct AL Apache Licence Header?
- [X] Are there Doxygen comments in the headers?
- [ ] Are any new features, behaviour changes documented in the .md format [documentation](https://github.com/AnimalLogic/AL_USDMaya/docs)?
- [X] Have you added, updated tests to cover new features and behaviour changes?
- [X] Have you filled out at least one changelog entry?

### CODE REVIEW

**Review by robthebloke — APPROVED** [review]
lgtm

**Comment by elrond79** [comment]
Done!

File: translators/pxrUsdTranslators/tests/testPxrTranslators.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        
+        tempFile = tempfile.NamedTemporaryFile(
+            suffix=".usda", prefix="AL_USDMayaTests_exportProxyShape_", delete=True)
+        
+        mc.createNode("transform", n="world")
+        mc.createNode("transform", n="geo", p="world")
+        
+        # create one proxyShape with a time offset
+        mc.select(clear=1)
+        proxyShapeNode = mc.AL_usdmaya_ProxyShapeImport(file="/mnt/Samsung/luma/rez-build/al_usdmaya/dev/src/translators/pxrUsdTranslators/tests/sphere2.usda")[0]
```
[the remark this answers is not in the document]
Comment (fabal) [reply translators/pxrUsdTranslators/tests/testPxrTranslators.py]: Don't worry about this, I'll fix it internally as we would like to merge it today.

[the remark this answers is not in the document]
Comment (elrond79) [reply translators/pxrUsdTranslators/tests/testPxrTranslators.py]: Doh - sorry I didn't catch that myself!

**Pull request merged**